src/javaRosa.js

```diff
@@ -1452,9 +1331,10 @@ define([
         loadXML: function (xmlString) {
             var _this = this,
                 langs = this.opts().javaRosa.langs,
-                Itext;
+                Itext, itextMap;
 
             this.data.javaRosa.Itext = Itext = new ItextModel();
+            this.data.javaRosa.itextMap = itextMap = {};
```

Comment:
This should only be an issue if they added the itext manually and were planning to add appropriate mugs later in vellum, correct?  Seems like a pretty small edge case to me